product/views.py

```diff
@@ -1,3 +1,26 @@
-from django.shortcuts import render
+import json
 
-# Create your views here.
+from django.views     import View
+from django.http      import JsonResponse
+
+from user.models    import User 
+from product.models import Product
+
+class SearchView(View):
+    def get(self, request):
+        result = request.GET['result']
+        results = Product.objects.filter(title__contains=result)
```

Comment:
```suggestion
        keyword = request.GET['result']
        products = Product.objects.filter(title__contains=keyword)
```
보다 직관적인 변수명을 사용해주시는 것이 좋습니다